test/integration/features/provision_various_versions.feature

```diff
@@ -0,0 +1,29 @@
+@provision-various-versions @openshift @minishift-only
+Feature: Provision all major Openshift versions
+  As a user I can provision major versions of openshift
+
+Scenario Outline:: Provision all major OpenShift versions
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --openshift-version <serverVersion>" succeeds
+     Then Minishift should have state "Running"
+     When executing "minishift openshift version" succeeds
+     Then stdout should contain
+     """
+     openshift <serverVersion>
+     """
+      And JSON config file "machines/minishift.json" contains key "OcPath" with value matching "<OcVersion>"
+     When executing "minishift oc-env" succeeds
+     Then stdout should contain
+      """
+      <OcVersion>
+      """
+Examples:
+    | serverVersion | OcVersion |
+    | v1.5.1        |  v3.6.0   |
+    | v3.6.0        |  v3.6.0   |
+    | v3.7.0        |  v3.7.0   |
```

Comment:
Since we agreed to have all major supported version then shouldn't we also add `1.4.1` here?

Reply:
@LalatenduMohanty This will fail as each time it will provision --openshift-version on same minishift instance. To avoid the failure  need to add ```executing "minishift delete --force" succeeds``` at the end of the scenario.

This would be like:
```
@provision-various-versions @openshift @minishift-only
Feature: Provision all major Openshift versions
  As a user I can provision major versions of openshift

Scenario Outline:: Provision all major OpenShift versions
    Given Minishift has state "Does Not Exist"
     When executing "minishift start --openshift-version <serverVersion>" succeeds
     Then Minishift should have state "Running"
     When executing "minishift openshift version" succeeds
     Then stdout should contain
     """
     openshift <serverVersion>
     """
      And JSON config file "machines/minishift.json" contains key "OcPath" with value matching "<OcVersion>"
     When executing "minishift oc-env" succeeds
     Then stdout should contain
      """
      <OcVersion>
      """
     Then executing "minishift delete --force" succeeds
      And Minishift should have state "Does Not Exist"
Examples:
    | serverVersion | OcVersion |
    | v1.5.1        |  v3.6.0   |
    | v3.6.0        |  v3.6.0   |
    | v3.7.0        |  v3.7.0   |
```

Reply:
Added v1.4.1